components/Main/Exercises/Exercises.js

```diff
@@ -16,39 +18,47 @@ export default function Exercises() {
     }
   }
 
+  function handleChange(event) {
+    setSearchTerm(event.target.value);
+  }
+
   return (
     <StyledExercises>
       <h2>Biceps Exercises</h2>
-      {dataBiceps.map(({ name, equipment, difficulty, instructions, id }) => (
-        <li key={id}>
-          <span>
-            <h3>{name.toUpperCase()}</h3>
-
+      <SearchBar onChange={handleChange} />
+      {dataBiceps
+        .filter((item) => {
+          if (searchTerm == "") {
+            return item;
```

Comment:
why do you want to return the item?  of the searchterm is an empty string, it can't include anything, no?

Reply:
I am returning item/exercise, because if the searchTerm is empty all exercises should be displayed. By returning null the exercises are not displayed.